tests/e2e/scenarios/upgrade-ab/run-test.sh

```diff
@@ -52,7 +52,7 @@ KOPS_B=$(kops-download-from-base)
 
 KOPS="${KOPS_B}"
 
-"${KOPS_B}" set cluster "${CLUSTER_NAME}" "cluster.spec.kubernetesVersion=${K8S_VERSION_B}"
+"${KOPS_B}" edit cluster "${CLUSTER_NAME}" "--set=cluster.spec.kubernetesVersion=${K8S_VERSION_B}"
```

Comment:
We use the master branch's scenario scripts for testing all kops versions, so this change is causing failures in kops 1.21 jobs:

https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/e2e-kops-aws-upgrade-k120-ko120-to-k121-ko121/1416173182599041024

Reply:
As I see it, we have three alternatives:

* Add "kops set" back in as a hidden command until we drop 1.21 as a destination for upgrade tests
* Make the script change the command based on the version of the destination kops
* Use the release branch's scenario script